_posts/events/2018-11-22---developing-next-get-web-apps.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: single
+title: "2018-11-22 - Developing a NextGen .NET Core WebApp with ASP.NET Boilerplate framework"
+date: 2018-11-22 13:37:00 +0000
+comments: false
+published: true
+categories: ["events"]
+tags: ["Events"]
+alias: ["/events/2018-11-22-developing-next-get-web-apps"]
+author: Tom Kerkhove
+redirect_from:
+ - /events/2018-11-22-developing-next-get-web-apps.html
+---
+
+## Agenda
+
+### Developing a NextGen .NET Core WebApp with ASP.NET Boilerplate framework
+In this session we would like to present a case study made for a hospital in Belgium. The WebApp is a reservation and planning system for operation room's and surgeons. It prevents double bookings of the OR's and will automatically cancel reservations when surgeons are taking holidays. Technologies used are .NET Core, Angular 4, RabbitMQ and IdentityServer 4, all of this is running on Azure.
+
+The focus in this sessions will be on the use of the ASP.NET Boilerplate framework. The framework allowed us to do Rapid Application Development and to have a very solid base framework that uses all of the patterns we find in modern development. As an extra we would like to briefly show also the use of Retrace and Stackify on Azure. Those are the debugging and monitoring tools we used for this WebApp.
+
+This session is a technical deep-dive for architects and developers as we will look in detail to all the code.
+
+<img src="/assets/media/speakers/wim-van-den-broeck.png" alt="Wim Van den Broeck" align="left" height="100" width="100" style="margin-right: 20px;">**Speaker:** *Wim is a .NET Consultant for 2commit with 25 years of software development experience with almost 20 years of experience in .NET and C#. He has written half a dozen articles on C# and .NET programming. He is specialized in the implementation of client/server, database, graphics and/or internet/based systems using the Visual Studio .NET suite. In the past few years his focus lies mainly on the Azure cloud platform.*
+
+<br />
+
+### TBA
+Second slot to be announced.
+
+<br />
+
+## Practical details
+
+**Event date:** November 22, 2018 - you are welcome from 18:00, session starts 18:30
+
+**Event location:**<br />
+<img width="120" height="60" align="right" alt="" src="/assets/media/sponsors/logo-2commit.png">2Commit<br />
```

Comment:
@KrisvanderMast Can you please verify this? Wasn't sure if this was the location.

Reply:
Can you ping me via mail when this is known? Will then get Pretix set up.